src/style.css

```diff
@@ -0,0 +1,116 @@
+/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-right: 50px;
+  margin-bottom: 48px;
+  cursor: pointer;
```

Comment:
why on the entire card? to me it makes sense to have on the title, image and button